pkg/api/annotation_test.go

```diff
@@ -0,0 +1,24 @@
+package api_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/pyroscope-io/pyroscope/pkg/model"
+	"github.com/pyroscope-io/pyroscope/pkg/service"
+)
+
+type mockService struct {
+	createAnnotationResponse func() (*model.Annotation, error)
+}
+
+func (m *mockService) CreateAnnotation(ctx context.Context, params service.CreateAnnotationParams) (*model.Annotation, error) {
+	return m.createAnnotationResponse()
```

Comment:
@kolesnikovae  we seem to be using `gomock` for the other tests in this package, should we keep using it moving forward?

Reply:
It would be nice. However, I think it's ok to use a manually crafted mock if it is simpler and more convenient in a particular case. I was just too lazy to manually mock a dozen of methods